src/LiveComponent/doc/index.rst

```diff
@@ -2453,6 +2453,40 @@ The parent-child system is *smart*. And with a few tricks
 (:ref:`such as the key prop for lists of embedded components <rendering-quirks-with-list-of-embedded-components>`),
 you can make it behave exactly like you need.
 
+.. _passing-blocks:
+
+Passing Blocks to Components
+----------------------------
+
+Passing blocks to Live components works completely the same way you would `pass blocks for Twig Components`_. Except...
```

Comment:
Passing content via blocks to Live...

Also, the next paragraph is a bit thick. We know the technical word is "context", users think "variables". Suggestion to replace the entire paragraph with:

... Except with one important difference: when a component is re-rendered, any variables defined
only in the "outside" template will note be available. For example, this won't work:

Reply:
yep agreed, changed